### ISSUE TITLE
[BUG] Media object page not navigable by TAB or SHIFT+TAB

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5406]
# Description

The [Avalon accessibility page](https://samvera.atlassian.net/wiki/spaces/AVALON/pages/1957953669/Accessibility) documents that TAB and SHIFT+TAB should be available to move through the interactive elements of the page. 

# To Reproduce

Load any media object page page in avalon-dev (or the Ramp demo site). If you hit TAB, nothing happens. If you try this on a media object page in MCO, tab will move you through the links on the page, beginning with the 'Skip to main content' link that's hidden at the top of the page. 

Note that other pages in Avalon seem to be working ok with tab navigation. Most likely an issue on the Ramp side. Tab may be getting captured from other code designed to make sure keyboard events that should be on the player/Ramp elements aren't causing action on undesired parts of the page.

# Done Looks like

Page navigation via keyboard should work as described on the [Avalon accessibility page] (https://samvera.atlassian.net/wiki/spaces/AVALON/pages/1957953669/Accessibility) for the media object page:

- [x] `TAB`: move forward through each interactive element on the page (links, text fields, buttons, etc.).
- [x] `SHIFT+TAB`: move backward through each interactive element.
- [x] `ENTER`: trigger an element's behavior (e.g. pressing `ENTER` on a link will open the link in the browser).
- [x] When an element is focused it will be surrounded by an outline to enhance its visibility.
- [ ] When using `TAB`, the first element of each page is a link titled "Skip to main content". When viewing an audio or video page, the "Skip to main content" link shifts the browser focus directly to the `Play` button.
- [x] The Stream Quality selector allows the user to `TAB` between available stream qualities and pressing `ENTER` to enable the desired quality.
- [ ] Users are able to navigate into the Details / Transcripts / File tabs via keyboard (maybe hitting enter on a tab title?)

**Comment by masaball** [comment]
Ramp #267 will need to be merged and a new build brought in before being able to test and confirm full functionality in Avalon is back.

**Comment by joncameron** [comment]
I think this is ready to close. For the two unchecked done criteria:
- "Skip to main content" does not seem necessary any longer; the first active element is the player now. I believe this was implemented in the past because it used to take a long time to cycle through elements before a user would hit the player for play/pause functionality, but that's not the case anymore. The player should be first in the order.
- Ramp should cycle through the tabs in the Nav UI for MetadataDisplay/SupplementalFiles/Transcript so each can be selected to display within that area, and those should be next in line after the StructuredNav component. I'll create a new issue for this.

Selecting the Next button in the media player also causes Ramp to lose context of the current tab index causing the user to have to tab through again from the beginning, which is another bug I noticed while testing that I'll create an issue for.

**Comment by joncameron** [comment]
Created https://github.com/samvera-labs/ramp/issues/273 for focus on tab labels and https://github.com/samvera-labs/ramp/issues/274 for retaining focus context after events.

**Comment by elynema** [comment]
@joncameron If we move the admin buttons back up above the player, does skip to main content still seem appropriate?

**Comment by joncameron** [comment]
It's still appropriate and this was my misunderstanding about the location and purpose of 'skip to main content'. It's not that we should pass over it within the item page content area/Ramp, it's that it exists at the very top level of the page and should be updated to link to the Ramp media player element, instead of `#content`. 

ERB line: https://github.iu.edu/AvalonIU/Avalon/blob/7e3a0a36b915d0f3ced981c926a3edce02f41cc7/app/views/layouts/avalon.html.erb#L43

Issue: https://github.com/avalonmediasystem/avalon/issues/5440
